lib/ansible/playbook/play_context.py

```diff
@@ -443,130 +377,30 @@ def set_task_and_variable_override(self, task, variables, templar):
 
     def make_become_cmd(self, cmd, executable=None):
         """ helper function to create privilege escalation commands """
+        # DEPRECATED
 
-        prompt = None
-        success_key = None
-        self.prompt = None
+        if cmd and self.become:
 
-        if self.become:
+            # load/call become plugins here
+            plugin = become_loader.get(self.become_method)
```

Comment:
I don't understand why we'd keep any of this here anymore, as it could all be done later when the become_loader is used in the connection plugin. With this, we're pretty much just double-loading the plugin (with caching maybe we don't care) but it's just very disjointed, and trying to find where this is done when all the other become stuff has moved out of PlayContext seems confusing.

Reply:
its a backwards compatibility issue, i kept in case 3rd party code still calls this, none of our current code does so it won't double load for us.

Reply:
it was also a good way to run existing unit tests